### DISCUSSION TITLE
help upgrading from 3.10 to 4.0.1-hotfix

### DISCUSSION THREAD

**Discussion by beavis69** [Novik/ruTorrent#2441] [Q&A]
Hello,

What is the php version recommended for ruTorrent 4.0.1, wiki page still says server needs php 5 !?

Is there a way to upgrade from 3.10 to 4.0.1-hotfix without losing my current configuration ?
I've tried to upgrade, and ratio group / autotools / rss configuration was lost.

I've tried restoring share and conf directories, but i still lose all settings...

Thanks

**Comment by halilnevzat** [comment]
same here, im using rutorrent on my router, there should be a single click update version automation. i was doing it by manually backing up my conf files and doing a manual replacements.

**Comment by beavis69** [comment]
I've retried today, and its working.

There are differences in old config.php and new config.php : 
* some constants became variables
* $profilePath has changed from `$profilePath = '../share';` to` $profilePath = '../../share';`

So i edited the new config.php and old config.php side by side, and now my settings are not lost from version 3.10.
